git-filter-repo

```diff
@@ -1406,7 +1408,7 @@ class FastExportParser(object):
         self._parse_literal_command()
       elif self._currentline.startswith(b'done'):
         if self._done_callback:
-          self._done_callback()
+          self._done_callback(self._callback_data)
```

Comment:
Oh, ick, this can't be the second argument because there isn't a first argument because there's nothing to operate on.  Hmm...also, does it even make sense for the user data in the done callback?  I'm not sure, because it's just such a rare callback to want or need.